content/release-notes/1.25.0.md

```diff
@@ -0,0 +1,24 @@
+---
+date: 2020-12-09T10:30:14-08:00
+linktitle: "1.25.0"
+title: "1.25.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<features>}}
+* Added the ability to control access to the KOTS Admin Console by integrating with an identity provider. This feature is only available for licenses that have the Identity Service feature enabled.
```

Comment:
This is interesting. These notes are read both by end customers and by vendors. This statement is true, but vendors will only see that feature enabled if they're flagged in. I think we need to mention that. Also, we should call out that it's experimental or alpha.